docs/design/coreclr/botr/clr-abi.md

```diff
@@ -79,6 +81,8 @@ However, unlike native varargs, all floating point arguments are not promoted to
 
 Managed varargs are not supported in .NET Core.
 
+Managed/native varargs are currently not supported on RISC-V.
```

Comment:
```suggestion
Managed/native varargs are supported on Windows only.
```

Reply:
There is no RISC-V version of Windows, so no RISC-V support falls out of that.

Reply:
Done